mergeSTR/README.md

```diff
@@ -1 +1,24 @@
-UNDER CONSTRUCTION
\ No newline at end of file
+# MergeSTR 
+
+MergeSTR is a tool for merging vcf files from GangSTR. 
+
+## Usage 
+To run mergeSTR use the following command: 
+```
+./mergeSTR.py \
+  --vcfs <vcf file1, vcf file2> \
+  --out test
+  [additional options]
+```
+
+Required Parameters: 
+* **--vcf <GangSTR VCF>** VCF file output by GangSTR. Can be unzipped or bgzipped, must be indexed.
+* **--out <string>** prefix to name output files
+
+Special Merge Options: 
+* **`--update-sample-from-file`**: Uses file names, rather than sample header names, when merging
+* **`--merge-ggl`**: Merge GGL fields 
```

Comment:
write next to this (Under construction) since we haven't finished implementing this yet